frontend/src/hooks/useTable.tsx

```diff
@@ -36,26 +36,45 @@ export const useTable = (route: Program | undefined) => {
   const tabs = [
     { index: 0, label: StatusNames.ACTIVE, value: Status.ACTIVE },
     { index: 1, label: StatusNames.INACTIVE, value: Status.INACTIVE },
-    { index: 2, label: StatusNames.PENDING, value: Status.PENDING },
+    { index: 2, label: 'Pending Approval', value: Status.PENDING },
```

Comment:
Eh?

Reply:
It's just the label being changed in one place (as per figma) I didn't want to change the enum since "Pending" is  correct, except that on the tabs it is displayed differently now.